ui.content/src/main/content/jcr_root/content/dam/formsanddocuments/hclecomm/user-complaint-form/_jcr_content/renditions/cq5dam.thumbnail.48.48.png.dir/.content.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@rakeshkumar-verma : hope we are not using default thumbnails but specific to hcl branding

Reply:
@rishabh-behl  Included hcl branding thumbnails